uber/automated_emails.py

```diff
@@ -8,7 +8,7 @@
 class AutomatedEmail:
     instances = OrderedDict()
 
```

Comment:
This is the change for #1271 so that emails require approval by default.  As you can see below, we explicitly mark a couple of emails as not needing approval, mostly the payment confirmation emails.  Other than that we'll need to be explicit about what we send, which seems like a good thing.